README.md

```diff
@@ -976,7 +976,7 @@ See **[Ticketing](#ticketing)**
   * [Buddy GO](https://buddy.works/) - The Git and Continous Integration / Delivery Platform. `Fair` `Nodejs/Java`
   * [Concourse](https://concourse.ci/) - Concourse is a CI tool that treats pipelines as first class objects and containerizes every step along the way. ([Demo](https://ci.concourse.ci/), [Source Code](https://github.com/concourse/concourse)) `Apache` `Go`
   * [drone](https://drone.io/) - Drone is a Continuous Delivery platform built on Docker, written in Go. ([Source Code](https://github.com/drone/drone)) `Apache` `Go`
-  * [Factor.io](https://factor.io/) - It's like IFTTT (if-this-then-that) for Dev and DevOps. ([Source Code](https://github.com/factor-io/factor)) `MIT` `Ruby`
+  * [Factor](https://github.com/factor-io/factor) - It's like IFTTT (if-this-then-that) for Dev and DevOps. `MIT` `Ruby`
```

Comment:
The website is still available, just at http://www.factor.io/ now instead of https://factor.io -- Can you redo this to just replace the original URL with the new one.